src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -5464,5 +5486,39 @@ void N()
                 Punctuation.CloseCurly,
                 Punctuation.CloseCurly);
         }
+
+        [Theory]
+        [CombinatorialData]
+        [WorkItem(45807, "https://github.com/dotnet/roslyn/issues/45807")]
+        public async Task FunctionPointer(TestHost testHost)
+        {
+            var code = @"
+class C
+{
+    delegate* unmanaged[Stdcall, SuppressGCTransition] <int, int> x;
+}";
+
+            await TestAsync(code,
+                testHost,
+                Keyword("class"),
+                Class("C"),
+                Punctuation.OpenCurly,
+                Keyword("delegate"),
+                Operators.Asterisk,
```

Comment:
How do we classify the * in a regular pointer like `int*` in unsafe code? Is it an operator like this or something else?

Reply:
(surprisingly to me) It's an operator, just like this.